benefits/in_person/views.py

```diff
@@ -130,6 +134,7 @@ def enrollment(request):
             "form_server_error": tokenize_server_error_form.id,
             "form_success": tokenize_success_form.id,
             "form_system_error": tokenize_system_error_form.id,
+            "title": f"{agency.long_name} | In-person enrollment | {admin_site.site_title}",
```

Comment:
I agree with @lalver1 that we want to add this to the context for the error view functions below (`reenrollment_error`, etc.).

Currently, they have no page title:
![image](https://github.com/user-attachments/assets/86085e3f-cd2a-4cc9-b7f5-2173ca3c815d)